nxrm-ha/README.md

```diff
@@ -320,6 +321,244 @@ Azure Key Vault is disabled by default. If you would like to store your database
+  
+       ```
+         serviceAccount:
+           enabled: true
+           name: nexus-repository-deployment-sa
+           labels: {}
+           annotations:
+             meta.helm.sh/release-name: <your-release-name>
+             meta.helm.sh/release-namespace: nexusrepo
+             iam.gke.io/gcp-service-account: <service-account-name>@<project-id>.iam.gserviceaccount.com <- your GCP project service account e-mail
+        
+       ```
+
+   2. Enable ingress 
+        ```
+        ingress:
+          name: "nexus-ingress"
+          enabled: true
+        ```  
+   3. Update ingress properties to correspond to GKE controller class https://cloud.google.com/kubernetes-engine/docs/how-to/load-balance-ingress#create-ingress 
+        ```
+        ingress:
+          name: "nexus-ingress"
+          enabled: true
+          ...
+          defaultRule: true
+          additionalRules: null
+          ingressClassName: gce
+          ...
+          annotations:
+            kubernetes.io/ingress.class: "gce"
+        ```
+   4. Enable Nexus NodePort service 
+    
+         ```
+         service:  #Nexus Repo NodePort Service
+           annotations: {}
+           nexus:
+             enabled: true
+         ```
+
+   5. Now you can update secrets separately (follow steps 6, 7, 8 below) 
+      OR install ESO ([External Secret Operator]https://external-secrets.io/latest/introduction/getting-started/)
+
+        ```
+        helm install external-secrets \
+           external-secrets/external-secrets \
+            -n external-secrets \
+            --create-namespace
+        ```
+      and use external secrets from Google Secret Manager (https://external-secrets.io/latest/provider/google-secrets-manager/)
+   
+
+   6. Enable db secret 
+        ```    
+        secret:
+          secretProviderClass: "secretProviderClass"
+          provider: provider # e.g. aws, azure etc
+          dbSecret:
+            enabled: true # Enable to apply database-secret.yaml which allows you to specify db credentials
+          db:
+            user: nxrm
+            userAlias: nxrm
+            password: nxrm
+            passwordAlias: nxrm
+            host: 10.10.0.3
+        ```
+
+   7. (Optional) Set default password for Nexus instances 
+        ```
+          nexusAdminSecret:
+            enabled: true # Enable to apply nexus-admin-secret.yaml which allows you to the initial admin password for nexus repository
+            adminPassword: admin123 #You should change this when you login for the first time
+        ```
+
+   8. Set license (use base64 to encode file)  
+        ```
+          license:
+            name: test-users.lic
+            licenseSecret:
+              enabled: true
+              fileContentsBase64: cylwwtYx6Fg1CUa9yGqBuhGhgc4IS67Ha/+uvxSpA  == your base64 encoded license file == Gd7Z3+WS/0LIeugxSIa+ZDqtg7AR+U3d9ZJA==
```

Comment:
You can also specify the license file with your helm command as in:

--`set-file secret.license.licenseSecret.file=<path to your license file>`

Reply:
added comment in doc